travis/get_openapi_description.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# python3
+
+import os
+
+from absl import app
+
+from grr_response_server.gui import api_call_router
+from grr_response_server.gui.api_plugins import metadata as metadata_plugin
+
+OPENAPI_JSON_FOLDER_NAME = os.environ.get(
+  "OPENAPI_JSON_FOLDER_NAME",
+  default="generated_description"
+)
+OPENAPI_JSON_FILE_NAME = os.environ.get(
+  "OPENAPI_JSON_FILE_NAME",
+  default="openapi_description.json"
+)
+HOME_PATH = os.environ["HOME"]
```

Comment:
I don't think this variable buys us much.

Reply:
You're right. Thank you for pointing this out! I removed it and used `os.environ["HOME"]` directly.